modules/ROOT/pages/_partials/flex-tls-cipher.adoc

```diff
@@ -0,0 +1,66 @@
+//tag::connectedSelect[]
+== Select Ciphers
+
+When you configure a TLS context, default ciphers are applied to it based on the TLS version you selected. In addition to the defaults, you can select other ciphers to use with the selected TLS version. Each TLS context can have multiple ciphers.
+
+To select ciphers:
+
+. Click *Ciphers* to select available ciphers. 
```

Comment:
I'm not familiar with this UI but would it be more accurate to say "_Click *Ciphers* and select one of the available ciphers_" ? Otherwise, this seems like two different steps: 
. Click *Ciphers* to show the available ciphers. 
. Select a cipher.